src/content/docs/security/new-relic-security/security-bulletins/security-bulletin-nrsg2202-01.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: "New Relic advises updating .NET Agent for customers employing Microsoft Extensions Logging with log forwarding enabled"
```

Comment:
`Agent` s/be lowercase (not a release blocker)

Reply:
Is this the doc title? it's very long.

Reply:
Yes it is, but it's the title they requested.